cmd/snap-confine/snap-confine.c

```diff
@@ -63,6 +64,26 @@ void sc_maybe_fixup_permissions()
 	}
 }
 
+// sc_maybe_fixup_udev will remove incorrectly created udev tags
+// that cause libudev on 16.04 to fail with "udev_enumerate_scan failed".
+// See also:
+// https://forum.snapcraft.io/t/weird-udev-enumerate-error/2360/17
+void sc_maybe_fixup_udev()
+{
```

Comment:
I would _probably_ prefer a simple list of  calls to`unlinkat(tag_dir_fd, "+nvidia-foo");` (there are about 4 or 5 of files to unlink) that is less sophisticated and operates only on the tags affecting the snap we are dealing with but I don't have strong opinions about it.